db/schema.rb

```diff
@@ -126,10 +126,14 @@
   create_table "user_settings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.boolean  "receive_notification",            default: true
     t.integer  "unread_notification"
-    t.float    "radius_display",       limit: 24, default: 5.0
+    t.boolean  "favorite_location",               default: false
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/ExtraSpacing
**Severity**: INFO
**File**: db/schema.rb L129

Unnecessary spacing detected.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*